src/integration-test/tests/citizen/claimantResponse/steps/claimant-reponse.ts

```diff
@@ -148,8 +190,8 @@ export class ClaimantResponseSteps {
 
   acceptPartAdmitFromBusinessWithAlternativePaymentIntention (): void {
     taskListPage.selectTaskViewDefendantResponse()
-    defendantsResponsePage.submit()
-    defendantsResponsePage.submitHowTheyWantToPay() // bug
+    viewDefendantsResponsePage.submit()
+    viewDefendantsResponsePage.submitHowTheyWantToPay() // bug
```

Comment:
What does the comment `bug` mean?

Reply:
I couldn't find the reason in the history. I'll remove it